internals-js/src/validate.ts

```diff
@@ -254,6 +254,12 @@ class Validator {
           sourceASTs(field.appliedDirectivesOf('deprecated')[0], field)
         ));
       }
+      if (field.defaultValue !== undefined && !isValidValue(field.defaultValue, field, new VariableDefinitions())) {
```

Comment:
`undefined` means "there is not default value" so it felt right to not validate. And in fact, `isValidValue` currently deal with `undefined` as a value like it does for `null` and so it would fail for a non-nullable type. To be fair, we could probably change the handling of `undefined` in `isValidValue` so it works, but I think it's more explicit to have the code say "there is no default, no validation to do".

Reply:
got it. Thanks.